src/index.jsx

```diff
@@ -4,17 +4,37 @@ import App from './App';
 import 'bootstrap/dist/css/bootstrap.min.css';
 import './App.scss';
 import ThemeProvider from './components/ThemeProvider';
+import UserProvider from './components/UserProvider';
 
 const axe = require('@axe-core/react');
 
+const USER_DATA = [
+  {
+    id: 0,
+    name: 'Jo',
+    email: 'jo@email.com',
+    password: 'password123',
+    homeCity: 'New York',
+  },
+  {
+    id: 1,
+    name: 'Coco',
+    email: 'coco@email.com',
+    password: 'password123',
+    homeCity: 'Frankfurt',
+  },
+];
+
```

Comment:
I think this is good enough for the purposes of our project. Were you planning on allowing the user to login with their own account, or just sticking to hard-coded users? I'm not really expecting to just because that means we have to utilitze a database or something to store that info.

Reply:
It would have been nice to have as a feature, but I think it could be a future/stretch goal. I think that implementation would require something relatively in-depth?

Reply:
Yeah it would definitely require some planning to integrate. I don't think it would be difficult since using a sql database isn't hard but rather just time-consuming  to hook up to our React project. Just because I don't see a way to do it with just React states any built in React tools. For now, this looks good!